client/src/components/Profile.jsx

```diff
@@ -0,0 +1,143 @@
+                "password": this.state.new_password,
+                "id": this.state.user.id
+            }
+            fetch(server_url + '/profile/' + window.location.href.split("/").pop(), {
+                method: 'PUT',
+                headers: {'Content-Type':'application/json',
+                'Access-Control-Allow-Origin':'http://localhost:3000'},
+                body: JSON.stringify(data)
+            })
+            window.Materialize.toast("Profile Updated", 2500);
+          }
+          else {
+            window.Materialize.toast("Incorrect repeated password!", 2500);
+          }
+        }
+    }
+
+    onChangeHandler = (e) => {
+        let { id } = e.currentTarget
+        this.setState({ [id]: e.currentTarget.value })
+    }
+
+    handleClick_Main() {
+        this.setState({ edit: false });
+
+    }
+    handleClick_Edit() {
+        this.setState({ edit: true });
+    }
+
+    Avatar_image() {
+       if (this.state.user.avatar == null) {
+          return (
+            <img src="http://dialpharma.com/media/img/default_profile.png" alt="Default icon" />
+          );
+      }
+      else {
+         return (
+           <img src={this.state.user.avatar} alt="Avatar icon" />
+         );
+       }
+   }
+
+
+  render() {
+      if (this.state.edit === false) {
+          return (
+            <div>
+              <div>
+                <h1>Profile</h1>
+              </div>
+                <div id="main">
+                <Row>
+                    <Col s={6}>
+                        <Row>
+                            <this.Avatar_image />
+                        </Row>
+                    </Col>
+                    <Col s={2}>
+                        <Row s={1}><h4>Nickname:</h4></Row>
+                        <Row s={1}><h4>Email:</h4></Row>
+                        <Row s={1}><h4>First Name: </h4></Row>
+                        <Row s={1}><h4>Last Name: </h4></Row>
+                        <Row s={1}><h4>Location: </h4></Row>
+                    </Col>
+                    <Col s={3}>
+                        <Row s={1}><h4>{this.state.user.nickname}</h4></Row>
+                        <Row s={1}><h4>{this.state.user.email}</h4></Row>
+                        <Row s={1}><h4>{this.state.user.first_name}</h4></Row>
+                        <Row s={1}><h4>{this.state.user.last_name}</h4></Row>
+                        <Row s={1}><h4>{this.state.user.location}</h4></Row>
+                        <Row s={1}><Button waves='light' onClick={this.handleClick_Edit}>Edit</Button></Row>
+                    </Col>
+                </Row>
+                <br />
+              </div>
+            </div>
+        );
+      }
+      else {
+          return (
+            <div>
+              <div>
+                <h1>Profile</h1>
+              </div>
+                <Row>
+                    <Input s={12} id="email" label="Email" placeholder={this.state.user.email} onChange={this.onChangeHandler} />
```

Comment:
The input at Navbar have id="email" too. Id must be unique, you can name it smth like email_profile...